libraries/Microsoft.BotKit/BotkitDialogWrapper.cs

```diff
@@ -24,21 +25,26 @@ public BotkitDialogWrapper(DialogContext dialogContext, IBotkitConversationStep
         /// <summary>
         /// Gets or sets an object containing variables and user responses from this conversation.
         /// </summary>
+        /// <value> An object with variables. </value>
```

Comment:
I think that the whitespaces at the beginning and the End of the description should be removed, just for having consistency with the other tags. Should be something like this:
```
<value>Description.</value>
```